Description: Drop grpcio from requirements
 Not needed at distro level as we're in charge of
 out dependencies.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/requirements.txt
+++ b/requirements.txt
@@ -13,4 +13,3 @@ futures>=3.0;python_version=='2.7' or py
 futurist>=1.2.0 # Apache-2.0
 oslo.utils>=3.15.0 # Apache-2.0
 oslo.serialization>=1.10.0 # Apache-2.0
-grpcio>=1.0.0,<1.16.0
